lab-glen/.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
```

Comment:
add package lock until they properly update, maybe I need to update my npm again, who knows. Anyhow, great work